frontend/src/pages/pipelines/create/steps/BasicInformation.tsx

```diff
@@ -270,15 +284,15 @@ const BasicInformation: React.FC<BasicInformationProps> = (
               />
             ) : null
           }
-          errorText={
-            assetsS3BucketEmptyError ? t('pipeline:valid.s3BucketEmpty') : ''
-          }
+          errorText={getS3BucketErrorMessage()}
         >
           <Autosuggest
```

Comment:
I think changing the `Autosuggest` component to [**Select**](https://cloudscape.design/components/select/)  components would make it simpler."